tests/integration/suite/daprd/metrics/httpserver_highcardinality.go

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
Consider moving both http tests into a `http` directory. This improves test organisation.